src/input/types.js

```diff
@@ -14,6 +14,10 @@ import {
   ENHANCER_POSITION,
 } from './constants.js';
 
+export type ClearInputEventT<T> =
```

Comment:
what benefit do we gain from the generic `T`? will this ever not be called on an `HTMLInputElement`?

Reply:
tbh i am not sure 🤔 it seems like we default to a generic `T` for most event handlers though. `onChange?: (e: SyntheticInputEvent<T>) => mixed` for example. maybe there is a reason for those handlers but not this case? or perhaps it is just a legacy thing that is being repeated for no good reason.

Reply:
actually it could be called with `HTMLTextAreaElement` as well

Reply:
actually our textarea is just a big input so i guess not